topics/client-engines.md

```diff
@@ -132,7 +132,12 @@ This is the recommended Apache-based engine for new projects.
 
    ```kotlin
    ```
-   {src="snippets/_misc_client/Apache5Create.kt" include-lines="1-4,7-23"}
+   {src="snippets/_misc_client/Apache5Create.kt" include-lines="1-4,7-33"}
+
+   - Use `configureConnectionManager` to set maximum connections. This preserves Ktor-managed engine behavior and
```

Comment:
I'm not sure, but maybe we can rephrase it so it's not only about maximum connections, but also for all other connection manager properties?
And we aren't using `HttpTimeout` plugin in the example, so probably shouldn't mention "ensures proper timeout mapping"